packages/cloud/src/queries/service-logs.ts

```diff
@@ -1,14 +1,14 @@
 import type { CreateServiceLog, ServiceLogType } from '@logto/schemas';
 import type { PostgreSql } from '@withtyped/postgres';
 import { sql } from '@withtyped/postgres';
-import type { Queryable } from '@withtyped/server';
+import type { JsonObject, Queryable } from '@withtyped/server';
 
 import { insertInto } from '#src/utils/query.js';
 
 export type ServiceLogsQueries = ReturnType<typeof createServiceLogsQueries>;
 
 export const createServiceLogsQueries = (client: Queryable<PostgreSql>) => {
-  const insertLog = async (data: Omit<CreateServiceLog, 'payload'>) =>
+  const insertLog = async (data: Omit<CreateServiceLog, 'payload'> & { payload?: JsonObject }) =>
```

Comment:
What is the original payload type in CreateServiceLog?

Reply:
Also since don't have the context, why did we omit payload from the log previously, but added as an optional field now? Is there any logo content conversion?

Reply:
In @logto/schemas we defined `payload` as ArbitraryObject type and with-typed temporarily is incompatible with that, as you can see from the screenshot, sijie will fix this later on. Temporarily use `JsonObject` as workaround.
<img width="725" alt="image" src="https://user-images.githubusercontent.com/15182327/231929263-10875e83-ce69-42a1-b6a3-0e3fe4b8d539.png">

Reply:
Added in-place comments as TODO memo.